apps/channel/components/collaborators/new_collaborator_view.coffee

```diff
@@ -0,0 +1,82 @@
+Backbone = require "backbone"
+_ = require 'underscore'
+sd = require("sharify").data
+mediator = require '../../../../lib/mediator.coffee'
+analytics = require '../../../../lib/analytics.coffee'
+Collaborators = require '../../../../collections/collaborators.coffee'
+
+newCollaboratorTemplate = -> require('./templates/new.jade') arguments...
+inviteTemplate = -> require('./templates/invite.jade') arguments...
+resultsTemplate = -> require('./templates/results.jade') arguments...
+
+module.exports = class NewCollaboratorView extends Backbone.View
+
+  events:
+    'keyup .new-collaborator__search' : 'searchCollaborators'
+    'click .js-collaborator-result' : 'addCollaborator'
+    'click .js-collaborator-invite' : 'inviteCollaborator'
+
+  initialize: ({ @channel, @collaborators} )->
+    @render()
```

Comment:
I would try to avoid views that render inside of their initialize and just have the caller invoke it.